src/css/application.css

```diff
@@ -1 +1,279 @@
+.application-section .section-description {
+  max-width: 709px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.application-form {
+  max-width: 500px;
+  margin: 0 auto;
+}
+
+.form-group {
+  margin-bottom: 18px;
+}
+
+.form-group.comment {
+  margin-bottom: 28px;
+}
+
+.form-group .label {
+  display: none;
+}
+
+.form-input {
+  color: var(--heading-color);
+  border-radius: 12px;
+  border: 1px solid var(--form-border);
+  font-family: 'Manrope', sans-serif;
+  padding: 16px 18px;
+  width: 100%;
+}
+
+.form-input:focus {
+  border-color: var(--border-color);
+  outline: none;
+}
+
+textarea.form-input {
+  resize: none;
+}
+
+.teacher {
+  border: none;
+  margin: 0;
+  padding: 0;
+}
+
+.teacher .title {
+  color: var(--heading-color);
+  font-weight: 600;
+}
+
+.radiobuttons {
+  display: flex;
+}
+
+.option {
+  align-items: center;
+  display: flex;
+  gap: 8px;
+}
+
+input[type="radio"] {
+  appearance: none;
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  display: none;
+}
+
+.radio-label {
+  border: 2px solid var(--radio-border);
+  border-radius: 50%;
+  cursor: pointer;
+  position: relative;
+}
+
+.radio-label:after {
+  background-color: var(--button-background);
+  border-radius: 50%;
+  content: '';
+  display: none;
+  height: 10px;
+  left: 50%;
+  position: absolute;
+  top: 50%;
+  transform: translate(-50%, -50%);
+  width: 10px;
+}
+
+input[type="radio"]:checked + .radio-label {
+  border-color: var(--border-color);
+}
+
+input[type="radio"]:checked + .radio-label:after {
+  display: block;
+}
+
+.radiobutton-label {
+  cursor: pointer;
+}
+
+.form-submit {
+  align-items: center;
+  background-color: var(--button-background);
+  border: none;
+  border-radius: 12px;
+  color: var(--secondary-color);
+  cursor: pointer;
+  display: flex;
+  font-weight: 700;
+  justify-content: center;
+  margin-bottom: 16px;
+  transition: background-color 250ms cubic-bezier(0.4, 0, 0.2, 1);
+  width: 100%;
+}
+
+.form-submit:hover,
+.form-submit:focus {
+  background-color: var(--active-color);
+}
+
+.application .message {
+  color: var(--primary-color);
+  margin: 0;
+}
+
+.application .message .link {
+  color: inherit;
+}
+
+.application .message .link:hover,
+.application .message .link:focus {
+  text-decoration: none;
+}
+
+.application-image {
+  border-radius: 30px;
+  background-color: var(--substrate-color);
+  overflow: hidden;
+  width: 492px;
+}
+
+.application-image img {
+  display: block;
+  height: auto;
+  width: 100%;
+  clip-path: ellipse(70% 90% at 50% 7%);
+}
+
+.application-image .description {
+  color: var(--secondary-color);
+  font-size: 16px;
+  font-weight: 400;
+  margin: 0;
+  padding: 20px 20px 40px 20px;
+  position: relative;
+  text-align: center;
+}
+
+.application-image .description .number {
+  display: block;
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 1.2;
+  margin-bottom: 8px;
+  text-align: center;
+}
+
+@media screen and (max-width: 767px) {
```

Comment:
maybe we can use only a mobile-first approach?

Reply:
not to override values as it was specified in last theme or homework I don't remember

Reply:
The matter was to avoid rewriting styles twice when we start with mobile-first. So, if there are specific styles that change for tablet and desktop, for tablet they need to be defined in the range between 768 and 1280.

Reply:
It's fine by me if we won't get respond that in 'our lessons' description should be 665px or any other section

Reply:
I think we shouldn't worry about that :) Anyway our mentor will check our work before. And I think there were a reason for doing this "specific" length for description.

Reply:
I think there wasn't. If designer is using three colors that deffer 10% of transparency on single page it could be just his/hers 'vision'